doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -189,6 +189,22 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-9-0 v2-8-0
+
+DRGN-23363: A flash operation executed on the system workqueue might result in ``-ETIMEDOUT``
+  A flash operation executed on the system workqueue might result in ``-ETIMEDOUT`` if there is an active Bluetooth LE connection.
+
+  **Workaround:** Move flash operations to a different workqueue.
```

Comment:
True, `different thread` is better, thanks for the suggestion.

> btw. Isn't this issue related to `NCSDK-29354` ? (seems a bit like a duplicate)

Yeah, but according to the reason in `NCSDK-29354`, these are 2 different issues which result in the same behavior. I was able to reproduce the timeout even with `CONFIG_BT_CONN_TX_NOTIFY_WQ` enabled, which doesn't exclude the fact that the issue explained in `NCSDK-29354` `happens because Bluetooth RX context waits for the connection TX notify that is done in the system workqueue context`.

I think I'd keep both known issues.